pkg/asset/manifests/azure/cloudproviderconfig.go

```diff
@@ -37,15 +37,30 @@ func (params CloudProviderConfig) JSON() (string, error) {
 			// ref: https://github.com/kubernetes/kubernetes/blob/4b7c607ba47928a7be77fadef1550d6498397a4c/staging/src/k8s.io/legacy-cloud-providers/azure/auth/azure_auth.go#L69
 			UserAssignedIdentityID: "",
 		},
-		ResourceGroup:                params.ResourceGroupName,
-		Location:                     params.GroupLocation,
-		SubnetName:                   params.SubnetName,
-		SecurityGroupName:            params.NetworkSecurityGroupName,
-		VnetName:                     params.VirtualNetworkName,
-		VnetResourceGroup:            params.NetworkResourceGroupName,
-		RouteTableName:               params.ResourcePrefix + "-node-routetable",
-		CloudProviderBackoff:         true,
-		CloudProviderBackoffDuration: 6,
+		ResourceGroup:          params.ResourceGroupName,
+		Location:               params.GroupLocation,
+		SubnetName:             params.SubnetName,
+		SecurityGroupName:      params.NetworkSecurityGroupName,
+		VnetName:               params.VirtualNetworkName,
+		VnetResourceGroup:      params.NetworkResourceGroupName,
+		RouteTableName:         params.ResourcePrefix + "-node-routetable",
+		CloudProviderBackoff:   true,
+		CloudProviderRateLimit: true,
```

Comment:
There is a note in [RP docs](https://github.com/Azure/ARO-RP/blob/7ccf7d654fc3865a68afb8fe33df60a9b648aeb4/docs/upstream-differences.md):

>CARRY: HACK: disable CloudProviderRateLimit
> Enabling CloudProviderRateLimit seems to get us into serious problems with the LoadBalancer reconciliation logic.

Reply:
Ok, missed that. But looks like this is getting us into another kind of problems. V3 Ci is broken as V4 clusters are exhausting all ARM limits